lib/Remotery.h

```diff
@@ -729,6 +734,15 @@ typedef struct rmtProperty
 // Reset all RMT_PropertyFlags_FrameReset properties to their default value
 #define rmt_PropertyFrameResetAll() _rmt_PropertyFrameResetAll()
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+rmtU32 rmt_HashString32(const char* s, int len, rmtU32 seed);
```

Comment:
Move this to the private section right at the bottom.

I'll move all the private property stuff down later as well, when I get over my current task.

Reply:
As a private function, should it also be prefixed with "_"?

Reply:
Hmmm... yes.